lib/fake_stripe.rb

```diff
@@ -33,8 +33,8 @@ def self.stub_stripe
     Stripe.api_key = 'FAKE_STRIPE_API_KEY'
     FakeStripe.reset
     stub_request(:any, /api.stripe.com/).to_rack(FakeStripe::StubApp)
+    stub_request(:any, /js.stripe.com/).to_rack(FakeStripe::StubApp)
```

Comment:
Seems like this should be `FakeStripe::StubStripeJS` instead of `FakeStripe::StubApp`, but AFAIK, this stub doesn't intercept anyway.